code/modules/food_and_drinks/food/snacks_pastry.dm

```diff
@@ -219,6 +219,16 @@
 	tastes = list("pretzel" = 1)
 	foodtype = GRAIN | SUGAR
 
+/obj/item/reagent_containers/food/snacks/dogtreat
+	name = "dog treat"
+	desc = "For good boys only."
```

Comment:
Pretty sure a majority of fur characters are female, so this is sexist as fuuuuuck.

But I could be wrong